functions/bouncer/src/middleware/stripeEventQueue.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-console */
+import debug from 'debug';
+import { dataSources } from '@thatconference/api';
+import { Firestore } from '@google-cloud/firestore';
+import pubSub from '../gcp/pubSub';
+
+const dlog = debug('that:api:functions:bouncer:middleware:stripeEventQueueMw');
+const historyStore = dataSources.cloudFirestore.history;
+const firestore = new Firestore();
```

Comment:
thinking we should again set this at the root like I mentioned in bouncer so we only create it once.

Reply:
Not a connection at this point, only a reference to the function. We could consolidate any such references to a request reference if you feel it will be better. It is something I have not done so don't have a good opinion on. Being a function reference I don't see it being 'expensive'.

Reply:
I am assuming that you are referring to `const firestore = new Firestore();` and not `historyStore` which is how I viewed this feedback initially.

Reply:
Refactored in brinks: ac099c2